js/creator-23.js

```diff
@@ -322,7 +322,7 @@ const setSymbolAliases = new Map([
 const mana = new Map();
 // var manaSymbols = [];
 loadManaSymbols(['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20',
-				 'w', 'u', 'b', 'r', 'g', 'c', 'x', 'y', 'z', 't', 'untap', 's', 'oldtap', 'originaltap', 'purple', "inf", "alchemy"]);
+				 'w', 'u', 'b', 'r', 'g', 'c', 'x', 'y', 'z', 't', 'untap', 's', 'oldtap', 'originaltap', 'purple', "inf", "alchemy", "pawprint"]);

```

Comment:
Move loading the symbol into the next line alongside 'e' and 'a'. The true at the beginning of the call on that line will make it so the color of the symbol matches the surrounding text, becoming white for borderless cards and so on.

Reply:
I also would prefer that the symbol is named 'p' matching the symbol in the rules text and scryfall, to simplify importing, with the old 'p' being renamed to 'h'.